### ISSUE TITLE
mRNA representation

### ISSUE THREAD

**Issue by Hrovatin** [lbcb-sci/RiNALMo#14]
After reading the paper I am unsure if this model can be directly used for mRNA representation without fine-tuning.
Namely, the training data contains no mRNAs (if I understand correctly). However, the model was then FT on mRNA tasks.
1.) May I ask why mRNAs were not used in pre-training - what is the reason for that? 
2.) Did you check masking performance on different sequence types including mRNAs? Similar as for structure prediction, although mRNAs were also missing there.

**Comment by saldanB** [comment]
Don't know if it can help but [here](https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-025-60872-5/MediaObjects/41467_2025_60872_MOESM4_ESM.pdf) there's the full nature-comm revision: in particular reviewer 1 in comment 3 could be interesting for your question.
